src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/PolicyCmdletBase.cs

```diff
@@ -106,5 +148,97 @@ protected JToken GetObjectFromParameter(string parameter)
                 ? JToken.FromObject(FileUtilities.DataStore.ReadFileAsText(path))
                 : JToken.FromObject(parameter);
         }
+
+        /// <summary>
+        /// Gets the resource Id from the supplied PowerShell parameters.
+        /// </summary>
+        protected string MakePolicyAssignmentId(string scope, string resourceName)
+        {
+            return ResourceIdUtility.GetResourceId(
+                resourceId: scope,
+                extensionResourceType: Constants.MicrosoftAuthorizationPolicyAssignmentType,
+                extensionResourceName: resourceName);
+        }
+
+        /// <summary>
+        /// Returns the policy or policy set definition resource Id given management group name and/or subscription id and/or policy definition name
+        /// </summary>
+        private string MakeDefinitionId(string managementGroupName, Guid? subscriptionId, string policyDefinitionName, string providerName)
```

Comment:
"providerName" is the fully qualified resource type (i.e. Microsoft.Authorization/policyDefinitions), right?  Lets change it to fullyQualifedResourceType or something similar.  providerName would mean just "Microsoft.Authorization"

Reply:
Done